meta-lmp-bsp/conf/machine/include/ultrazed-lmp.inc

```diff
@@ -55,3 +55,7 @@ PMU_FIRMWARE_DEPLOY_DIR ?= "${DEPLOY_DIR_IMAGE}"
 
 # Not building for QEMU standard Xinix build
 EXTRA_IMAGEDEPENDS_remove = "qemu-devicetrees"
+
+# bitstream support
+MACHINE_FIRMWARE_append_uz3eg-iocc = " bitstream-extraction fpga-manager-script"
```

Comment:
Since this is machine specific, move this line to conf/machine/uz3eg-iocc.conf instead.

Also, please split the machine changes into a different commit.

Reply:
will do